src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,30 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { jwtService } from '../services/index.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'];
+  const throwUnauthorizedError = () => {
+    throw ApiError.unauthorized({
+      user: 'unauthorized user',
+    });
+  };
+
+  if (!authorization) {
+    throwUnauthorizedError();
+  }
+
+  const [, token] = authorization?.split(' ');
```

Comment:
The use of optional chaining (`?.`) is unnecessary here since `authorization` is already checked for existence. You can safely use `authorization.split(' ')`.